### PR TITLE
iina: init at 1.2.0

### PR DESCRIPTION
**Pull request by arkivm**
###### Motivation for this change
iina: init at 1.2.0

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/iina/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+stdenv.mkDerivation rec {
+  pname = "iina";
+  version = "1.2.0";
+
+  src = fetchurl {
+    url = "https://github.com/iina/iina/releases/download/v${version}/IINA.v${version}.dmg";
+    sha256 = "sha256-kbh+gAVfCXoct6jJGXnetTAzFfIGdVLL5zh/SL/EJzY=";
+  };
+
+  buildInputs = [ undmg ];
```
Comment (SuperSandro2000) [inline pkgs/applications/video/iina/default.nix:16]:
```suggestion
  nativeBuildInputs = [ undmg ];
```

**Review by SuperSandro2000 — COMMENTED** [review]
Not sure if this should be under os-specific/darwin

**Comment by arkivm** [comment]
> Not sure if this should be under os-specific/darwin

I was thinking that too. But I saw some other packages that are darwin specific not under `os-specific/darwin` (e.g., `pkgs/applications/terminal-emulators/iterm2/`). So, I am not entirely sure. I will go with your recommendation

**Comment by arkivm** [comment]
bump

**Comment by SuperSandro2000** [comment]
@ofborg eval
@ofborg build iina

**Comment by SuperSandro2000** [comment]
```
copying path '/nix/store/xhnd28gwlq3a5v223largs3bymppba1a-lzfse-1.0' from 'https://cache.nixos.org'...
building '/nix/store/rh213mrk5qrrxhxfr5cj380qh0yl0n6j-IINA.v1.2.0.dmg.drv'...
copying path '/nix/store/n8dzlkj13q1wamcrfkzj383n288iig1m-undmg-1.1.0' from 'https://cache.nixos.org'...
trying https://github.com/iina/iina/releases/download/v1.2.0/IINA.v1.2.0.dmg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   652  100   652    0     0   1902      0 --:--:-- --:--:-- --:--:--  1900
100 62.4M  100 62.4M    0     0  20.0M      0  0:00:03  0:00:03 --:--:-- 30.6M
building '/nix/store/xfpy9dpgqk35baj4q0pv3amhz7p8vwci-iina-1.2.0.drv'...
unpacking sources
unpacking source archive /nix/store/ax842p33s60al6h0v1iq3q7d6cw93813-IINA.v1.2.0.dmg
unpacker produced multiple directories
builder for '/nix/store/xfpy9dpgqk35baj4q0pv3amhz7p8vwci-iina-1.2.0.drv' failed with exit code 1
error: build of '/nix/store/xfpy9dpgqk35baj4q0pv3amhz7p8vwci-iina-1.2.0.drv' failed
```

**Comment by arkivm** [comment]
oops. sorry.
fixed by adding `sourceRoot`. it was also creating `Applications` symlink.

**Pull request merged**